ionoscloud/resource_nfs_cluster.go

```diff
@@ -0,0 +1,206 @@
+package ionoscloud
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"slices"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	ionoscloud "github.com/ionos-cloud/sdk-go-nfs"
+
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services/nfs"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/utils"
+)
+
+// ValidNFSLocations is a list of valid locations for the Network File Storage Cluster.
+var ValidNFSLocations = []string{"de/fra", "de/txl", "qa/de/txl"}
```

Comment:
I think this should be in the client file, it would make more sense to find it there if you were to search for it.

Reply:
what do you think about `qa/de/txl` should we leave this location in?

Reply:
for the time being, yes, but it should be removed before the PR is merged into master

Reply:
then I'll take it out.